src/components/Formulier.jsx

```diff
@@ -0,0 +1,142 @@
+import "./Formulier.css";
+import {useState} from "react";
+
+function Formulier() {
```

Comment:
Een component zonder properties is niet echt een component. Hoe kan je nu dit formulier hergebruiken maar dan met bijvoorbeeld andere waardes? Niet. Je zou ook van de inputs een component kunnen maken die je vervolgens overal in je project kan gebruiken en aanpassen met properties indien nodig.

Verder maak je helemaal geen gebruik van dit component!